block/config.ml

```diff
@@ -1,6 +1,23 @@
 open Mirage
 
-let main = foreign "Unikernel.Main" (console @-> block @-> job)
+type shellconfig = ShellConfig
+let shellconfig = Type ShellConfig
+
+let config_shell = impl @@ object
+    inherit base_configurable
+    method configure i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "dd if=/dev/zero of=%s/disk.img count=100000" dir
+    method clean i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "rm %s/disk.img" dir
+    method module_name = "Functoria_runtime"
```

Comment:
Yes. In those cases, my usual practice is to use "whichever module name is define and mildly relevant".

We could define an empty module and use it for all those cases. Arguably we should have a way to express that, though ...